projects/dotcms-field-elements/src/components/dot-form/services/dot-upload.service.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { DotUploadService } from './dot-upload.service';
+
+describe('DotUploadService', () => {
+    function FormDataMock() {
+        this.append = jest.fn();
+    }
+
+    const globalAny: any = global;
+    globalAny.FormData = FormDataMock;
+
+    const fetchMock = jest.fn();
```

Comment:
Is a good idea to move this to the utils so we can import the mock when we need it in other tests.